### PR TITLE
[Crane] Fix Map zoom controls not working bug

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt [hunk 1]
```diff
@@ -192,15 +195,14 @@ private fun MapViewContainer(
     latitude: String,
     longitude: String
 ) {
+    val cameraPosition = remember(map) { LatLng(latitude.toDouble(), longitude.toDouble()) }
```
Comment (JoseAlcerreca) [inline Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt:198]: This is a good opportunity to showcase `rememberSaveable`?

Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt:198]: That'd be a micro-optimization IMO, as this is not a state to remember, just a constant I don't want to compute multiple times.

**Review by JoseAlcerreca — APPROVED** [review]

File: Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt (hunk 1 above)
Comment (chrisbanes) [inline Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt:198]: This won't update if `latitude` or `longitude` change 🤔

Comment (manuelvicnt) [reply Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt:198]: And `map` doesn't make any sense there! Thanks. Let's try to write correct code! Updating...

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**